docs/flyte_agents/developing_agents.md

```diff
@@ -159,29 +159,15 @@ For flytekit versions `>v1.10.2`, use `pyflyte serve agent`.
 
 ### 4. Deploy Your Flyte agent
 
-1. Update the FlyteAgent deployment's [image](https://github.com/flyteorg/flyte/blob/master/charts/flyteagent/templates/agent/deployment.yaml#L35)
+Update the FlyteAgent deployment's [image](https://github.com/flyteorg/flyte/blob/master/charts/flyteagent/templates/agent/deployment.yaml#L35)
 
 ```bash
 kubectl set image deployment/flyteagent flyteagent=ghcr.io/flyteorg/flyteagent:latest
 ```
 
-2. Update the FlytePropeller configmap:
-
-```YAML
- tasks:
-   task-plugins:
-     enabled-plugins:
-       - agent-service
-     default-for-task-types:
-       - bigquery_query_job_task: agent-service
-       - custom_task: agent-service
-```
-
-3. Restart FlytePropeller:
-
-```
-kubectl rollout restart deployment flytepropeller -n flyte
-```
+:::{note}
+Please make sure your propeller's image is >= 1.13.0 so that the propeller can automatically fetch the supported task types from your agent deployment.
```

Comment:
```suggestion
Please make sure your propeller's image version is >= 1.13.0 so that the propeller can automatically fetch the supported task types from your agent deployment.
```